third-party/download-content-shell/README.md

```diff
@@ -0,0 +1,8 @@
+# Download Content Shell
```

Comment:
awesome 👍 

- should this be `third-party/devtools-frontend/`? (that's usually the naming used)
- needs a LICENSE file parallel to this file

Reply:
added license

I don't think the folder needs to be the same name as the project when the code taken is very specific.

CDT happens to do this but I don't know of others https://source.chromium.org/chromium/chromium/src/+/master:third_party/devtools-frontend/src/third_party/i18n/